storage/store.go

```diff
@@ -643,6 +644,10 @@ func (s *Store) BootstrapRange() error {
 	if err := engine.MVCCPutProto(batch, ms, key, now, nil, zoneConfig); err != nil {
 		return err
 	}
+	// Range Tree setup.
+	if _, _, err := s.SetupRangeTree(batch, ms, now, desc.RaftID); err != nil {
```

Comment:
If this is the only place where SetupRangeTree is called we should probably get rid of the two unused return values.